src/3rd_party/pathie-cpp/src/path.cpp

```diff
@@ -902,7 +902,7 @@ Path Path::pwd()
  */
 Path Path::exe()
 {
-#if defined(__linux__)
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
BTW these changes are in an external library. Is there possibly an update that already does this? If not, are we going to contribute back?

Reply:
I have already contributed this upstream about a year ago: https://github.com/Quintus/pathie-cpp/pull/11

It seems that the marian version has diverged quite a bit since then...